packages/ember-glimmer-template-compiler/tests/precompile-test.js

```diff
@@ -0,0 +1,16 @@
+import { precompile, compile, template } from 'ember-glimmer-template-compiler';
+
+QUnit.module(`Glimmer Precompile:`);
+
+QUnit.test('returns a string', (assert) => {
+  let str = precompile('Hello');
+  assert.equal(typeof str, 'string');
+});
+
+QUnit.test('when wrapped in a template, precompile is the same as compile', (assert) => {
+  // Simulating what happens in a broccoli plugin
+  // when it is creating an AMD module
+  let precompiled = template(precompile('Hello'));
```

Comment:
we should assert that precompile returns a string, and that when it is evaluated, it returns what template() expects.